x/protorev/types/token_pair_arb_routes.go

```diff
@@ -26,29 +26,14 @@ func (tp *TokenPairArbRoutes) Validate() error {
 
 	// Iterate through all of the possible routes for this pool
 	for _, route := range tp.ArbRoutes {
-		if len(route.Trades) != 3 {
-			return fmt.Errorf("there must be exactly 3 trades in a route")
-		}
-
-		// In denoms must match either osmo or atom
-		if route.Trades[0].TokenIn != AtomDenomination && route.Trades[0].TokenIn != OsmosisDenomination {
-			return fmt.Errorf("the first trade must have either osmo or atom as the in denom")
+		if len(route.Trades) < 3 {
```

Comment:
Does something break if this is < 2? If we go generalized n-hops for the route then I think we should also support 2 hops

Reply:
Committed changes that allow for hops of length > 2. Added some stateless testing for this but we should add more in depth stateful testing in the follow up pr.